### PR TITLE
Keyboard auto Y/N

### PR DESCRIPTION
**Pull request by b0balagurak**

### CODE REVIEW

File: src/view/TerminalInterface.py [hunk 1]
```diff
@@ -247,7 +247,23 @@ def getConfirmation(self, message, okStr="Y", nokStr="N"):
...
-        choice = str(self.__getUserInput())
+
+        while True:
+            if okStr == "scratch":
+                choice = str(self.__getUserInput())
+                break
+            elif keyboard.is_pressed("Y"):
+                choice = "y"
+                keyboard.press("backspace")
+                time.sleep(0.2)
```
Comment (nrhoopes) [inline src/view/TerminalInterface.py:258]: ? should this also be in the next elif block?

File: src/view/TerminalInterface.py [hunk 2]
```diff
@@ -247,7 +247,23 @@ def getConfirmation(self, message, okStr="Y", nokStr="N"):
...
+                print(" " + "y") 
+                break
+            elif keyboard.is_pressed("N"):
+                choice = "n"
+                keyboard.press("backspace")
+                print(" " + "n")
+                break
```
Comment (nrhoopes) [inline src/view/TerminalInterface.py:265]: Do we need an else for if the user wants to abort? Or am I thinking about a different place for that?

**Review by nrhoopes — DISMISSED** [review]
See comments on file

File: src/view/TerminalInterface.py (hunk 1 above)
Comment (b0balagurak) [reply src/view/TerminalInterface.py:258]: it shouldnt be im sure

**Review by nrhoopes — APPROVED** [review]

**Pull request merged**